dts/bindings/sensor/ti,ina23x.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2021 The Chromium OS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+description: |
+    TI INA230 and INA231 Bidirectional Current and Power Monitor.
+    The <include/dt-bindings/sensor/ina23x.h> file should be included
+    in the DeviceTree and it provides Macros that can be used for
+    initializing the Configuration register.
+
+compatible: "ti,ina23x"
+
+include: i2c-device.yaml
+
+properties:
+    config:
+      type: int
+      required: true
+      description: Configuration register
+
+    current_lsb:
+      type: int
+      required: false
+      default: 1
+      # default of 1mA for current LSB
+      description: |
+        Value of Current LSB in milliamps. When set to 1mA
+        Current is read in mA, Bus Voltage in mV, Shunt
+        Voltage in uV, and Power in mW. Any other value
+        results in Current, Voltage, and Power registers
+        being read in counts.
+
+    rshunt:
+      type: int
+      required: true
+      description: Shunt resistor value in milliohms
+
+    mask:
+      type: int
+      required: false
+      default: 0
+      # default all alert sources to disabled
+      description: Mask register
+
+    alert_limit:
+      type: int
+      required: false
+      default: 0
+      # default alert limit is 0V
```

Comment:
The use of `default` must be justified in the description here also.

Reply:
Done